modules/core/src/v2/coins/trx.ts

```diff
@@ -249,6 +275,200 @@ export class Trx extends BaseCoin {
     return Buffer.from(sig, 'hex');
   }
 
+  /**
+   * Converts an xpub to a compressed pub
+   * @param xpub
+   */
+  xpubToCompressedPub(xpub: string): string {
+    if (!this.isValidXpub(xpub)) {
+      throw new Error('invalid xpub');
+    }
+
+    const hdNode = utxoLib.HDNode.fromBase58(xpub, this.bitcoinEncoding());
+    return hdNode.keyPair.__Q.getEncoded(false).toString('hex');
+  }
+
+  compressedPubToHexAddress(pub: string): string {
+    const byteArrayAddr = bitgoAccountLib.Trx.Utils.getByteArrayFromHexAddress(pub);
+    const rawAddress = bitgoAccountLib.Trx.Utils.getRawAddressFromPubKey(byteArrayAddr);
+    return Buffer.from(rawAddress).toString('hex').toUpperCase();
+  }
+
+  xprvToCompressedPrv(xprv: string): string {
+    if (!this.isValidXprv(xprv)) {
+      throw new Error('invalid xprv');
+    }
+
+    const hdNode = utxoLib.HDNode.fromBase58(xprv, this.bitcoinEncoding());
+    return hdNode.keyPair.d.toBuffer(32).toString('hex');
+  };
+
+  bitcoinEncoding(): any {
```

Comment:
Remove `any`, return type can be inferred.

Reply:
removed totally